inntekt/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/inntekt/InntektLøser.kt

```diff
@@ -87,25 +88,38 @@ class InntektLøser(
             response
         }
 
-    override fun onPacket(packet: JsonMessage, context: MessageContext) {
+    override fun onBehov(packet: JsonMessage) {
         measureTimeMillis {
             logger.info("Mottar pakke")
             sikkerLogger.info("Mottar pakke: ${packet.toJson()}")
-            val uuid = packet[Key.ID.str].asText()
+            val uuid = packet[Key.UUID.str].asText()
             logger.info("Løser behov $INNTEKT med id $uuid")
```

Comment:
er det mulig å legge til i loggen hva slags id dette er?
KEY.ID("@id") er slik jeg har forstått det iden som rapids og rivers rammeverket legger på alle meldinger som blir publisert.
Mens UUID er i dette tilfelle transaction id. Eller er det en annen id?

Reply:
UUID kan vare transaksjonsid hvis den er brukt i en statefull kontekst. Den kan også vare ID relatert til prossesen som er aktivert hvis konteksten er ikke statefull. 

@ID ikke brukes i det helle tatt på applikasjonsnivå så er det ikke noe poeng for oss å bry seg om det.